### ISSUE TITLE
Add pomatez to Flathub

### ISSUE THREAD

**Issue by gozes** [zidoro/pomatez#444]
Personally I much prefer flatpaks over appimages or snaps so would it be possible to add to Flathub?

**Comment by sekwah41** [comment]
Never done one of those, id need to take a look. Should be for the electron version at least as simple as setting another export target https://www.electron.build/configuration/flatpak.html

**Comment by sekwah41** [comment]
For the Tauri version i am working on it seems its not implemented yet https://github.com/tauri-apps/tauri/issues/3619 though may be possible to bundle ourselves.

**Comment by sekwah41** [comment]
Oh and also a action to upload to flathub

**Comment by gozes** [comment]
I think there is a GitHub action already for building and uploading it Flathub I think. Haven't looked at it in long while

**Comment by sekwah41** [comment]
One thing I've realised is that this would also likely make this available on the steam deck as they use Flathub for the discover apps section.

**Comment by gozes** [comment]
> One thing I've realised is that this would also likely make this available on the steam deck as they use Flathub for the discover apps section.

Yup you would be correct. Everything on the steam deck is flatpaks and trying to "sideload" anything from the package manager or snaps may get whipped on a future update

**Comment by gozes** [comment]
any updates?

**Comment by sekwah41** [comment]
Tauri seems to be able to push to flat pack and snapcraft now. Will Need to take a look at polishing up the Tauri version
